### PR TITLE
Added comaparison for package size

### LINKED ISSUE #6781: Highlight web3.js features [docs/articles]
Highlight web3.js bundle size by the different packages, in the docs and articles.
references:
- This can be added in the `Introduction` section of the `Getting started` category
- potentially research and add other advantages of web3.js over other libraries and add them

image

### PR DESCRIPTION
**Pull request by drparadox05**
## Description

This change resolves #6781 by adding an image to compare size of various bundles and including 1 more feature in the "Features of Web3.js v4" section of docs.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] I ran `npm run lint` with success and extended the tests and types if necessary.
- [ ] I ran `npm run test:unit` with success.
- [ ] I ran `npm run test:coverage` and my test cases cover all the lines and branches of the added code.
- [ ] I ran `npm run build` and tested `dist/web3.min.js` in a browser.
- [ ] I have tested my code on the live network.
- [ ] I have checked the Deploy Preview and it looks correct.
- [ ] I have updated the `CHANGELOG.md` file in the root folder.
- [ ] I have linked Issue(s) with this PR in "Linked Issues" menu.

### CODE REVIEW

File: docs/docs/guides/getting_started/introduction.md [hunk 1]
```diff
@@ -21,6 +21,9 @@ The following documentation will guide you through different use cases of Web3.j
 -    More efficient ABI Encoder & Decoder
 -    Custom Output formatters
 -    In compliance with Eth EL API
+-    Reduced build size of packages
+
+![Package Size Chart](PackageSizeChart.png)
```
Comment (jdevcs) [inline docs/docs/guides/getting_started/introduction.md]: @Deepak2030 we need comparison of web3 packages with other libs, 
**potentially research and add other advantages of web3.js over other libraries and add them**

@SantiagoDevRel web3 package size is changed with time so we will need to keep this updated!

Comment (drparadox05) [reply docs/docs/guides/getting_started/introduction.md]: I've updated the PR, let me know if it needs any improvement

File: docs/docs/guides/getting_started/introduction.md [hunk 2]
```diff
@@ -22,6 +22,8 @@ The following documentation will guide you through different use cases of Web3.j
 -    Custom Output formatters
 -    In compliance with Eth EL API
 
+![Comparison of various package sizes](PackageSizeComparison.jpg)
```
Comment (jdevcs) [inline docs/docs/guides/getting_started/introduction.md]: we should be showing (minified gzipped) sizes @Deepak2030 Thanks

Comment (drparadox05) [reply docs/docs/guides/getting_started/introduction.md]: Have updated, please check

**Comment by drparadox05** [comment]
Hey @jdevcs i've updated the PR with requested changes, please check

**Review by avkos — APPROVED** [review]

**Review by jdevcs — CHANGES_REQUESTED** [review]
web3 and other libs size change slightly in each release so this will need updates each time, I think this shouldn't part of doc

**Comment by jdevcs** [comment]
> web3 and other libs size change slightly in each release so this will need updates each time, I think this shouldn't part of doc

@Deepak2030 if you remove size image we can merge this PR

**Pull request closed without merging**